docs/tutorials/tutorial-sql-query-view.md

```diff
@@ -0,0 +1,168 @@
+---
+id: tutorial-sql-query-view
+title: "Tutorial: Use Query view"
```

Comment:
We can get rid of the convention of "Tutorial:" in the title. We can do this for the other tutorials later.
Also wondering if we can find a better title than "Use" This is about making the most of some of the built in features of the query view.  "Get to know the query view"? "Query view tips and tricks"? "Get the most out of the Query view?"

Reply:
Updated title